### PR TITLE
Feat: transpile packages

### PR DESCRIPTION
**Pull request by jamesseanwright**
* Move all package source code to `src` directories
* Transpile with Babel on `prepublishOnly` to `dist`
* Add `dev` entry point that points to `src` dir to support hot reloading in Storybook
* Move tests into `src` directory to preserve relative imports and DX (we should investigate running tests against transpiled code in the future)
* Updates Jest Configurator to only target `src` dir for coverage
* Updates CLI to generate components that follow this new structure

All of the unit tests pass, and the components render in both the web and native storybooks.

**Please note that**, due to the utils package now having an explicit entry point, the **tests no longer use fake timers by default**. This is because the `faketime` module was calling `jest.useFakeTimers()` upon import, which resulted in runtime errors outside of the test suite.

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/8986196a5ec76e0588f6c663bcfa1a4767fa4aec/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/c24f05d0b447eb086129403ea23cb8373f7cf61e/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/10776a272a0ea179a826b95fdf649e5c5bd50e00/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/59f51bbd63bfbc21463e0d27b5e8ddb87cfa09a7/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/50b1e4383fb763d766d3128b7c15aa566dcc6c79/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/2911eab795ca40ed104815f0fe2b07500e37a914/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/b0c715d29ec0c5a8cf7ded8a3b4a144d7c197776/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/f2aaccd2356af53ccb51334692283adbaabb3df2/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/89824a2591ffdf7b924300606384e7ea3c99f0b7/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/b4848c03b34d6c9dda83ef07ae4be0e82124221e/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/3c48286337343cb4a09e11cbf42f2016714c6def/index.html

File: packages/styleguide/styleguide.stories.styles.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const styles = {
   text: {
     textAlign: "center",
     fontFamily: fonts.body,
-    fontSize: fontSizes.caption,
+    fontSize: fontSizes.flags,
```
Comment (nikhedonia) [inline packages/styleguide/styleguide.stories.styles.js]: is this intentional?

Comment (jamesseanwright) [reply packages/styleguide/styleguide.stories.styles.js]: Definitely not! 😳 Good spot!

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/9f2dbf07fb83d6b4195c35a953830c50dae2eed8/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/f6d8fe92051c739370fb40bd63b1c973a0060df2/index.html

**Pull request merged**